.github/workflows/release_linux_x86_64.yml

```diff
@@ -98,3 +98,36 @@ jobs:
         export ORT_MAX_ML_OPSET_SUPPORTED_VERSION=3
         export ORT_MAX_ONNX_OPSET_SUPPORTED_VERSION=19
         pytest
+
+
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release' && github.event.action == 'published' && startsWith(github.ref, 'refs/tags/v') 
+    needs: [build]
+    steps:
+      - uses: actions/download-artifact@348754975ef0295bfa2c111cba996120cfdf8a5d
+        with:
+          name: wheels
+          path: dist
+
+      - name: Publish wheels to pypi
+        uses: pypa/gh-action-pypi-publish@2f6f737ca5f74c637829c0f5c3acd0e29ea5e8bf
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
+
+      - name: Create GitHub Release # TODO: notes?
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        run: >-
+          gh release create ''${{ github.ref_name }}' --repo '${{ github.repository }}' --notes ""
```

Comment:
I think we can assume a release has been created when this pipeline is triggered, because we use the release feature to create the tag.